ui/README.md

```diff
@@ -62,33 +97,12 @@ You can also run a script across all packages in the workspace using NPM's
 Note: If you are trying to run the app (`npm run start -w app`), you may need to
 first [setup a local backend server](https://github.com/perses/perses/blob/main/CONTRIBUTING.md).
 
-## Library Architecture Diagram
```

Comment:
I removed this mermaid diagram because it's out of date, we can't easily auto-generate it, and our packages are sufficiently complex that it's a pain to do by hand. I included a reference farther up how to get a job-based graph from turbo if someone wants/needs it on the fly.